src/main/msp/msp_rangefinder.c

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Is there any advantage to use this indirection? MSP 0x1F01 command can't be extended easily, so it can be directly linked to LIDARMT.